manifest.xml

```diff
@@ -27,7 +27,11 @@
   <rosdep name="facets" />
   <rosdep name="libxml2" />
   <rosdep name="pkg-config" />
-  <rosdep name="ruby-dev" />
+
+  <!-- rosdep key `ruby-dev` is not known to ROS.
+       Depend on `ruby` instead, which includes development files. -->
+  <!-- <rosdep name="ruby-dev" /> -->
```

Comment:
@doudou Where exactly is key `ruby-dev` defined in AutoProj? I could not find it in either [lib/autoproj/default.osdeps](https://github.com/rock-core/autoproj/blob/master/lib/autoproj/default.osdeps) or [package_set/rock.osdeps](https://github.com/rock-core/package_set/blob/b3412981e047c86b6615a55e826612c04ab2da02/rock.osdeps). Is it directly considered to be an OS package if not found elsewhere?

Reply:
It's in autoproj's `default.osdeps`. It is defined there, as autoproj itself needed to install it.